src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx

```diff
@@ -22,6 +25,7 @@ export const TableRelationsAccessible: React.FC<Props> = props => {
             <TableRelationElementOrigin
               relation={relation}
               originTable={table}
+              originTableRef={collectionRefs.current[table.id]}
```

Comment:
destinarionTableRef={collectionRefs.current[relation.toTableId]}

Reply:
Are you sure I need to change the prop name? I see it as the table in which is the origin of the relationship.

Wouldn't that be something like this?:
`originTableRef={collectionRefs.current[relation.toTableId]}`